pkg/api/norman/customization/cluster/validator.go

```diff
@@ -665,13 +666,23 @@ func validateGKENodePools(spec *v32.ClusterSpec) error {
 
 		version := np.Version
 		if version == nil || *version == "" {
-			errors = append(errors, fmt.Sprintf("node pool [%s] version cannot be empty", name))
+			versionErrors = append(versionErrors, fmt.Sprintf("node pool [%s] version cannot be empty", name))
 			continue
 		}
+
+		image := strings.ToLower(np.Config.ImageType)
+		// Windows images are WINDOWS_LTSC or WINDOWS_SAC. The cluster must have at least one non-Windows node pool.
+		if !strings.Contains(image, "windows") {
```

Comment:
Do we need to be concerned with the capitalization here? I noticed that your comment is in all caps.

Reply:
It's converted to lower case on the line above. Both upper and lowercase are valid. The UI sends it as uppercase.

Reply:
I'm sorry I missed that.